api/v1beta2/runtimecomponent_types.go

```diff
@@ -232,6 +235,12 @@ type RuntimeComponentService struct {
 	Bindable *bool `json:"bindable,omitempty"`
 }
 
+// RuntimeComponentNetworkPolicy configures the network policy
```

Comment:
These descriptions end up in the CRD and are also used in the OpenShift UI.
Suggestion: Defines the network policy

The child fields also require descriptions